_lessons/maui-and-mahuika/12-building-code-on-maui.md

```diff
@@ -135,7 +139,7 @@ SLURM_PARTITION=Debug mpiexec -n 6 simpleMPI
 
 Although the compiler drivers ```ftn```, ```cc``` and ```CC``` have a few options of their own, such as the ```-craype-verbose``` flag, they will pass through any additional compiler options to the underlying compiler. This means that you will still need to choose compiler flags that are specific to the Cray, Intel, or GNU compilers, and you will need to change them if you decide to switch compilers.
 
-For example, if you wanted to use the gfortran compiler, activate compiler warnings (```-Wall```), and require aggressive compiler optimisation (```-O3```), you would use the following commands:
+For example, if you wanted to use the gfortran compiler, activate compiler warnings (```-Wall```), and require aggressive compiler optimization (```-O3```), you would use the following commands:
```

Comment:
@mand35 the spelling here was correct before (i.e. with the 's' not 'z') - could you change back (and same for the others below)?